ceno_zkvm/src/scheme/verifier.rs

```diff
@@ -201,9 +201,10 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMVerifier<E, PCS>
         }
```

Comment:
Hi @darth-cy please refer new verifier logic here to support
one shard + one chip + multiple proofs + respective soundness check